### PR TITLE
Add async select to the share portfolio modal

### PR DESCRIPTION
**Pull request by lgalis**
Add async select to the share portfolio modal

Screenshot from 2019-12-09 14-58-57

### CODE REVIEW

**Comment by Hyperkid123** [comment]
@lgalis btw i might have possible fix for the overflowing context menu.

File: src/forms/form-fields/share-group-select.js [hunk 1]
```diff
@@ -34,6 +38,7 @@ ShareGroupSelect.propTypes = {
   inputName: PropTypes.string.isRequired,
   selectName: PropTypes.string.isRequired,
   groups: PropTypes.any,
```
Comment (Hyperkid123) [inline src/forms/form-fields/share-group-select.js]: This prop is no longer necessary

File: src/forms/form-fields/share-group-select.js [hunk 2]
```diff
@@ -34,6 +38,7 @@ ShareGroupSelect.propTypes = {
...
+  loadOptions: PropTypes.func,
```
Comment (Hyperkid123) [inline src/forms/form-fields/share-group-select.js]: This should be required

**Comment by lgalis** [comment]
@Hyperkid123 - feedback addressed

**Pull request merged**